generator/sidekick/refresh.go

```diff
@@ -26,6 +27,12 @@ import (
 // Reruns the generator in one directory, using the configuration parameters
 // saved in its `.sidekick.toml` file.
 func Refresh(rootConfig *Config, args []string) error {
+	fs := flag.NewFlagSet("refreshall", flag.ExitOnError)
```

Comment:
what's the reason for creating a new `flag.NewFlagSet` here?

typically I would initiate all of the flags in one place, so that running `go run ./sidekick -h` tells me all of the usage information